concepts.wip/arithmetic-operators/introduction.md

```diff
@@ -3,3 +3,26 @@
 Julia provides a number of [arithmetic operators](https://en.wikipedia.org/wiki/Arithmetic#Arithmetic_operations) for numeric types[^1].
 
 [^1]: You can find a list of them in the [Julia Manual](https://docs.julialang.org/en/v1/manual/mathematical-operations/#Arithmetic-Operators).
+
+Many of the arithmetic operations are straightforward. Below are examples of some operations in Julia that may have different **syntax** in other languages, ex: Python.
+
+```julia
+# Exponentiation
+julia> 3 ^ 2
+9
+```
+
+```julia
+# Integer Division
+julia> div(5, 2)
```

Comment:
This is about operators, so let's use the operator form even if they're equivalent:

```suggestion
# Integer Division
# ÷ can be typed as \div<tab>
julia> 5 ÷ 2
```

Reply:
Applied as per suggestion.